public/app/plugins/panel/graph/specs/series_override_ctrl.jest.ts

```diff
@@ -0,0 +1,43 @@
+import '../series_overrides_ctrl';
+import { SeriesOverridesCtrl } from '../series_overrides_ctrl';
+
+describe('SeriesOverridesCtrl', () => {
+  let popoverSrv = {};
+  let $scope;
+  let seriesOverridesCtrl;
+
+  beforeEach(() => {
+    $scope = {
+      ctrl: {
+        refresh: jest.fn(),
+        render: jest.fn(),
+        seriesList: [],
+      },
+      render: jest.fn(() => {}),
+    };
+    seriesOverridesCtrl = new SeriesOverridesCtrl($scope, {}, popoverSrv);
```

Comment:
Guess you can just do a `new SeriesOverridesCtrl($scope, {}, popoverSrv);` here and skip the assignment.